src/main/java/nl/tudelft/ti2206/group9/gui/skins/AndySkin.java

```diff
@@ -0,0 +1,17 @@
+package nl.tudelft.ti2206.group9.gui.skins;
+
+/**
+ * @author Maikel on 08/10/2015.
+ */
+public class AndySkin extends AbstractSkin {
+
+    /**
+     * Constructor for a Andy skin.
+     * @param price price
+     * @param name name
+     * @param location location in resources
+     */
+    public AndySkin(final int price, final String name, final String location) {
+        super(price, name, location);
```

Comment:
What is the point of creating six separate classes if they are all exactly the same? I would suggest "hardcoding" the price and location. In the current case, you can put the Plank skin in the `AndySkin` class, for example.
Suggested solution: `super(9999, "Andy", "andy");`
This can be done in the same way for all Skin classes.

<review,organization